cypress/integration/seededFlows/policyFlows/limitPostCreationToAdmins.spec.js

```diff
@@ -0,0 +1,27 @@
+/**
+   This is to verify visibility of buttons.  The authorization tests are
+   asserted in their corresponding policy tests.
+  */
+describe('Limit Post Creation to Admins', () => {
+  beforeEach(() => {
+    cy.testSetup();
+    cy.fixture('users/articleEditorV1User.json').as('user');
+    cy.enableFeatureFlag('limit_post_creation_to_admins');
+    cy.get('@user').then((user) => {
+      cy.loginAndVisit(user, '/');
+    });
+  });
+
+  it('clicking on User Avatar should open User Dropdown menu and no Create Post is visible', () => {
+    // TODO: If we go with the approach in this commit, then we should test the
+    // "Create Button" that's on the nav bar but not in the drop down.
+
+    cy.findByRole('button', { name: 'Navigation menu' }).as('menuButton');
+    cy.get('@menuButton')
+      .should('have.attr', 'aria-expanded', 'false')
+      .click()
+      .should('have.attr', 'aria-expanded', 'true');
```

Comment:
```suggestion
  cy.get('@menuButton')
      .should('have.attr', 'aria-expanded', 'false')
      .click()
      .should('have.attr', 'aria-expanded', 'true');

    cy.findByRole('link', {
      name: 'Article Editor v1 User @article_editor_v1_user',
    }).should('exist');

    cy.findByRole('link', { name: 'Create Post' }).should('not.exist');
```

I'm suggesting this change because initially when this menu loads, there's no 'Create post' link because the user data hasn't been loaded yet. We want to check that _after_ the user data has been handled, there's still no 'Create post' button, so we can use this additional check on the menu's user profile link to make sure we wait long enough for that to happen